SkietbaanBE/SkietbaanBE/Controllers/CompetitionController.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SkietbaanBE.Models;
+
+namespace SkietbaanBE.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/Competition")]
+
+    public class CompetitionController : Controller
+    {
+        private ModelsContext _context;
+        public CompetitionController(ModelsContext db)
+        {
+            _context = db;
+        }
+ //----------------------------------------------------------------------------------------------------------------------------------------
+        /** The method to return an array of competition objects that hold the true Status*/
+        // GET: api/Competition
+        [HttpGet]
+        public IEnumerable<Competition> Get()
+        {
+            //get the competitions where(Status == true)
+            var competitionIDsQuery = from Comp in _context.Competitions
+                                      where Comp.Status == true
+                                      select Comp;
+            List<Competition> competitionsIDs = competitionIDsQuery.ToList<Competition>();
+            // return _context.Competitions.ToArray<Competition>();
+            return competitionsIDs;
+        }
+//----------------------------------------------------------------------------------------------------------------------------------------
+        //Getting the competition by ID
+        // GET: api/Competition/5
+        [HttpGet("{id}")]
+        public async Task<Competition> CompetitionGet(int id)
```

Comment:
Its better practice to call this `GetCompetition`, makes it less confusing and keeps the standard of the rest of the project